test/scenarios-periodics/el93-crel@upgrade-from-el92-prel.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# Scenario tests upgrade from RHEL9.2 to RHEL9.3 (including MicroShift).
+
+start_commit="rhel-9.2-microshift-4.$(previous_minor_version)"
+target_commit=rhel-9.3-microshift-crel
+
+scenario_create_vms() {
+    if ! does_commit_exist "${target_commit}"; then
+        echo "Commit '${target_commit}' not found in ostree repo - skipping test"
+        return 0
+    fi
```

Comment:
I see this pattern repeated a few times. Maybe we should add an interface for the scenarios to report the images they require? I wonder if that could be optional.

Reply:
I would love that. I was actually thinking about this and it could be long-term solution for our image blueprints manual grouping